packages/ee/ui/embed-sdk/src/index.ts

```diff
@@ -37,6 +37,7 @@ type IframeWithWindow = HTMLIFrameElement & { contentWindow: Window };
 export const NEW_CONNECTION_QUERY_PARAMS = {
   name: 'pieceName',
   connectionName: 'connectionName',
+  date:'date'
```

Comment:
What is this i cant understand what date refer to? Where is it used? How is it used?